web/app/dashboard/dashboard.component.html

```diff
@@ -1,44 +1,44 @@
 <div class="fci-dashboard-header">
   <div fci-grid class="fci-dashboard-welcome">
-    <div fci-cell="6">
-      <div class="fci-dashboard-title"> Welcome to fastlane CI! </div>
+    <div fci-cell="12">
+      <div class="fci-dashboard-title"> Welcome to fastlane.ci! </div>
       <div class="fci-dashboard-subtitle">
-        Open source, self-hosted, mobile-optimized CI powered by fastlane brought to you by the fastlane team. </div>
+        Open source, self-hosted, mobile-optimized CI powered by fastlane brought to you by the fastlane team.
+      </div>
+      <button mat-raised-button color="primary">Add New Project</button>
     </div>
   </div>
 </div>
 
 <div fci-grid>
   <div fci-cell="12">
-    <mat-card class="fci-project-table">
-      <mat-progress-spinner mode="indeterminate" *ngIf="isLoading" class="fci-loading-spinner" diameter="30"></mat-progress-spinner>
-      <mat-table *ngIf="!isLoading" [dataSource]="projects">
-        <!-- Name Column -->
-        <ng-container matColumnDef="name">
-          <mat-header-cell *matHeaderCellDef> Project Name </mat-header-cell>
-          <mat-cell *matCellDef="let project">
-            <fci-status-icon [status]="project.latestStatus"></fci-status-icon>
-            <span>{{project.name}}</span>
-          </mat-cell>
-        </ng-container>
+    <mat-table class="fci-project-table" [dataSource]="projects">
+      <!-- Name Column -->
+      <ng-container matColumnDef="name">
+        <mat-header-cell *matHeaderCellDef> Project Name </mat-header-cell>
```

Comment:
spaces around `Project Name` probably not intentional, right?

Reply:
removed